waspc/cli/src/Wasp/Cli/Command/Requires.hs

```diff
@@ -0,0 +1,71 @@
+{-# LANGUAGE GeneralizedNewtypeDeriving #-}
+
+module Wasp.Cli.Command.Requires
+  ( -- * Using command requirements
+
+    -- This module exports the lower-level parts of the command requirement
+    -- API. To use these features to establish a requirement for a command,
+    -- use 'Wasp.Cli.Command.Common.commandRequires'. For convenience,
```

Comment:
Agreed, it would ideally be exported from `Wasp.Cli.Command`. Let's discuss this in the thread in `Wasp.Cli.Command.Common`: https://github.com/wasp-lang/wasp/pull/1218/files#r1217807700